pkg/apis/ceph.rook.io/v1/nfs.go

```diff
@@ -73,6 +73,23 @@ func (sec *NFSSecuritySpec) Validate() error {
 		if volSourceExistsAndIsEmpty(sidecar.SSSDConfigFile.VolumeSource) {
 			return errors.New("System Security Services Daemon (SSSD) sidecar is enabled with config from a VolumeSource, but no source is specified")
 		}
+
+		subDirs := map[string]bool{}
+		for _, additionalFile := range sidecar.AdditionalFiles {
+			subDir := additionalFile.SubPath
+			if subDir == "" {
+				return errors.New("System Security Services Daemon (SSSD) sidecar is enabled with additional file having no subPath specified")
```

Comment:
nit: The error messages don't need to spell out SSSD
```suggestion
				return errors.New("SSSD sidecar is enabled with additional file having no subPath specified")
```

Reply:
The other messages spell out `System Security Services Daemon`, so I think it makes sense to follow form here.